parlai/core/torch_agent.py

```diff
@@ -168,6 +172,7 @@ def map_valid(self, obs_batch, sort=True, is_valid=lambda obs: 'text_vec' in obs
         # set up the target tensors
         ys = None
         labels = None
+        y_lens = None
```

Comment:
this isn't used, right?

Reply:
confirm: I think this line is unnecessary

Reply:
Yes, it isn't used except within the if statement, will remove.